extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Metrics.tsx

```diff
@@ -1,87 +1,121 @@
+                  selectedTab === tab ? "active" : ""
+                }`}
+                onClick={() => {
+                  setSelectedTab(tab);
+                }}
+                key={tab}
+              >
+                {tab}
+              </div>
+            );
           })}
-          {dashboard?.rows?.filter((r: any) => !dashboard?.tabs?.includes(r.tab))?.length > 0 &&
+          {dashboard?.rows?.filter(
+            (r: any) => !dashboard?.tabs?.includes(r.tab)
+          )?.length > 0 && (
             <div
-              className={`application-metrics__Tab ${selectedTab === 'More' ? 'active' : ''}`}
-              onClick={() => { setSelectedTab('More') }}
-              key={'More'}
+              className={`application-metrics__Tab ${
+                selectedTab === "More" ? "active" : ""
+              }`}
+              onClick={() => {
+                setSelectedTab("More");
+              }}
+              key={"More"}
             >
               More
             </div>
-          }
+          )}
         </div>
-      }
+      )}
 
-      {!isLoading && dashboard?.rows && !dashboard?.rows?.filter((r: any) => dashboard?.tabs?.includes(r.tab) || selectedTab === 'More')?.length &&
-        <p>No charts assigned to the <strong>{selectedTab}</strong> tab.</p>
-      }
+      {!isLoading &&
+        dashboard?.rows &&
+        !dashboard?.rows?.filter(
+          (r: any) => dashboard?.tabs?.includes(r.tab) || selectedTab === "More"
+        )?.length && (
+          <p>
+            No charts assigned to the <strong>{selectedTab}</strong> tab.
+          </p>
+        )}
 
       {dashboard?.rows?.map((row: any) => {
-        if (dashboard?.tabs?.length && row?.tab !== selectedTab && !(!row?.tab && selectedTab === "More")) {
-          return <></>
+        if (
+          dashboard?.tabs?.length &&
+          row?.tab !== selectedTab &&
+          !(!row?.tab && selectedTab === "More")
+        ) {
+          return <></>;
         }
         return (
           <>
-            <div className='application-metrics'>
-              <span className='application-metrics__RowTitle'>
-                {row.title}
-              </span>
+            <div className="application-metrics">
+              <span className="application-metrics__RowTitle">{row.title}</span>
             </div>
-            <div className='application-metrics__ChartContainerFlex'>
+            <div className="application-metrics__ChartContainerFlex">
               {row?.graphs?.map((graph: any) => {
-                const url = `/api/extension/metrics/applications/${application_name}/groupkinds/${resource.kind.toLowerCase()}/rows/${row.name}/graphs/${graph.name}?name=${resourceName}.*&namespace=${namespace}&application_name=${application_name}&project=${project}&uid=${uid}&duration=${duration}`
+                const url = `/extensions/metrics/applications/${application_name}/groupkinds/${resource.kind.toLowerCase()}/rows/${
```

Comment:
this must be changed from:
`/extensions/metrics/applications/.......`
to 
`/extensions/metrics/api/applications/.......`

the same standard must me applied to all requests sent to the metrics-server

Reply:
Made the change.